cmd/juju/controller/addmodel.go

```diff
@@ -55,15 +55,22 @@ type addModelCommand struct {
 
 const addModelHelpDoc = `
 Adding a model is typically done in order to run a specific workload.
-To add a model, you must at a minimum specify a model name. You may
-also supply model-specific configuration, a credential, and which
-cloud/region to deploy the model to. The cloud/region and credentials
-are the ones used to create any future resources within the model.
 
+To add a model, you must specify a model name. 
 Model names can be duplicated across controllers but must be unique for
 any given controller. Model names may only contain lowercase letters,
 digits and hyphens, and may not start with a hyphen.
 
+You may also supply model-specific configuration, a credential, and which
+cloud/region to deploy the model to. The cloud/region and credentials
+are the ones used to create any future resources within the model.
+
+Credentials are always used when adding a model.
+If you are the admin user, your credentials will already be stored in the controller. 
```

Comment:
Not just the admin user. Any user who has updated credentials with the update-credential command. The text I used was just specific to the bug report, not so much in general.

Also, "Credentials are always used when adding a model" is not quite correct. It should be "credential" singular. But I don't think the sentence is required.

So I'd tweak the wording to reflect that if a controller already has a credential stored for the user creating the model, that credential will be used, else it will need to be specified. A credential is stored by either creating a model and using --credential, or running update-credential. In either case,  the named credential must previously have been set up using add-credential or autoload-credentials.